website/docs/cloud-docs/api-docs/state-versions.mdx

```diff
@@ -204,7 +233,11 @@ curl \
                 "created-at": "2021-06-08T01:22:03.794Z",
                 "size": 940,
                 "hosted-state-download-url": "https://archivist.terraform.io/v1/object/...",
+                "hosted-state-upload-url": "https://archivist.terraform.io/v1/object/...",
```

Comment:
If a state is finalized (or if it has a download URL), it won't have this upload URL in the response 🤔 

```suggestion
                "hosted-state-upload-url": null,
```